src/sentry/integrations/github/integration.py

```diff
@@ -833,75 +947,108 @@ def get_app_url(self) -> str:
 
     def dispatch(self, request: HttpRequest, pipeline: Pipeline) -> HttpResponseBase:
         with record_event(IntegrationPipelineViewType.GITHUB_INSTALLATION).capture() as lifecycle:
+            self.active_user_organization = determine_active_organization(request)
+
+            chosen_installation_id = pipeline.fetch_state("chosen_installation")
+            if chosen_installation_id is not None:
+                pipeline.bind_state("installation_id", chosen_installation_id)
+
             installation_id = request.GET.get(
                 "installation_id", pipeline.fetch_state("installation_id")
             )
             if installation_id is None:
                 return HttpResponseRedirect(self.get_app_url())
 
             pipeline.bind_state("installation_id", installation_id)
```

Comment:
if somebody passes `installation_id` with the request, would it override the `chosen_installation_id` in the pipeline state?

Reply:
please make this change